modules/sticky-table-headers/main/index.coffee

```diff
@@ -43,6 +43,21 @@ cloneEvents = (elem, clone) ->
       cloneChildren = clone.childNodes
       cloneEvents elemChildren[i], cloneChildren[i] for i in [0..elemChildren.length]
 
+getChildrenFromTable = (t, body, single) ->
+  realParents = getChildren(t.select(if body then 'tbody' else 'thead'), 'tr')
+  hx.flatten realParents.map (parent) -> getChildren(hx.select(parent), 'th, td', single)
+
+getChildren = (parent, selector, single) ->
+  children = if single
```

Comment:
Ah yes, I forgot about phantom.

I suppose since you are only checking the nodeType you _could_ get away with just filtering on `node.nodeName`, something like:

```
  children = Array.prototype.slice.call(parent.children).filter(node => node.nodeName === 'TBODY')
```

but it probably isn't worth changing now.

Its hard to tell if the last assumption would necessarily be the case without benchmarking it - my initial assumption was the opposite :-p 
But, this is something that can be investigated for #104 (ie best performance characteristics and best browser support). It probably depends a lot on what you are selecting